src/AlgebraicGeometry/Schemes/PrincipalOpenInclusion/Constructors.jl

```diff
@@ -0,0 +1,16 @@
+function complement(X::AbsSpec, h::Vector)
```

Comment:
I seem to recall that we have the function `hypersurface_complement` which does what your `complement` does?

Reply:
This is similar to `subscheme` vs. `sub`: One returns a map, the other doesn't.